File: src/main/java/io/renren/modules/api/annotation/AuthIgnore.java
package io.renren.modules.api.annotation;

import java.lang.annotation.*;

/**
 * api接口，忽略Token验证
 *
 * @author ChunLiang Hu
 * @email davichi2009@gmail.com
 * @date 2017-03-23 15:44
 */
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
@Documented
public @interface AuthIgnore {

}

File: src/main/java/io/renren/modules/api/annotation/AuthIgnore.kt
package io.renren.modules.api.annotation

/**
 * api接口，忽略Token验证
 *
 * @author ChunLiang Hu
 * @email davichi2009@gmail.com
 * @date 2017-03-23 15:44
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
@Retention(AnnotationRetention.RUNTIME)
@MustBeDocumented
annotation class AuthIgnore
